docs/book/src/topics/multitenancy.md

```diff
@@ -6,6 +6,11 @@ This is achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-id
 
 ## Identity Types
 
+### Workload Identity
+
+AAD pod identity is deprecated and workload identity is the next iteration of it.
```

Comment:
this contradicts the line right above (line 5) which says `this is achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-identity) library.`

I also think it's unclear which ones below are AAD pod identity and which ones aren't.

Reply:
Have tried to incorporate this comment. Let me know if it looks good or any more changes are required.

Reply:
I added some more specific comments with what I had in mind, let me know if it makes sense.

I really think the resulting docs need to be unambiguously pointing to workload identity and making it very clear that AAD pod identity is deprecated to avoid confusion.